parachain/primitives/core/src/types.rs

```diff
@@ -36,6 +37,22 @@ pub enum VerificationInput {
 		/// The index of the event within the block.
 		event_index: u32,
 	},
+	/// Light-client-based scheme checks receipt inclusion proof
+	/// TODO: use specialized structs, not byte arrays
+	LightReceipt {
+		// The receipt in the block
+		receipt: Vec<u8>,
+		// Merkle proof of inclusion in the block's receipts
+		proof: Vec<u8>,
+	},
+	/// Light-client-based scheme checks transaction inclusion proof
+	/// /// TODO: use specialized structs, not byte arrays
+	LightTransaction {
```

Comment:
I'm not sure if there's a use case for _only_ transaction inclusion proof, i.e. proving that a transaction was processed, but not proving its outcome

Reply:
yeh. there may be for some obscure uses but we don't need to implement this for now